Assets/Scripts/ShooterScript.cs

```diff
@@ -4,24 +4,13 @@
 
 public class ShooterScript : MonoBehaviour
 {
-    public PowerUpSystem powerUpSystem;
-
-    private Vector2 dirToMouse, raycast2DEndPoint, raycast2DHitPoint, bubbleSize;
-    [SerializeField] private GameObject raycast2DHitObject, bulletPrefab, bulletsParent, bulletSpawnPoint, gun;
-    public GameObject environmentPaintPrefab, environmentPaintParent;
-    private int bulletCollideLayer;
+    [HideInInspector] public PowerUpSystem powerUpSystem;
```

Comment:
Okay thats something i dont understand now. This branch is based on yesterdays main branch, how can the readme be so old? also if i update from main it says "this branch is already up to date." so im not entirly sure whats going on with the readme. to your second comment: it sounds like an overlook from my part, i will take a look at it, thx for the correction!
to the third commend: i just changed the collision on the physics2D tab, so that the bullets wont collide with other bullets. iam not sure what we intend here, this way it does look a bit smoother and makes testing easier. but as i said maybe we want the bullets to collide with them selfs, or maybe not.